packages/protocol/runTests.js

```diff
@@ -17,7 +17,7 @@ async function startGanache() {
     network_id: network.network_id,
     mnemonic: network.mnemonic,
     gasPrice: network.gasPrice,
-    gasLimit: 7000000,
+    gasLimit: 8000000,
```

Comment:
Unfortunately this will need to increase in several other places: scripts/bash/ganache.sh, scripts/devchain.ts.

We should unify this configuration eventually.